src/codeu/chat/client/commandline/Chat.java

```diff
@@ -14,22 +14,15 @@
 
 package codeu.chat.client.commandline;
 
-import java.time.format.DateTimeFormatter;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Scanner;
-import java.util.Stack;
-
-import codeu.chat.client.core.Context;
-import codeu.chat.client.core.ConversationContext;
-import codeu.chat.client.core.MessageContext;
-import codeu.chat.client.core.UserContext;
-import codeu.chat.util.Tokenizer;
+import codeu.chat.client.core.*;
```

Comment:
Tagging @iZake here to undo the import wildcard. It was likely added by someone's IDE automatically, but which we should still fix it.